Tests/WitsmlExplorer.Api.Tests/Workers/CopyLogWithParentWorkerTests.cs

```diff
@@ -17,45 +17,45 @@
 namespace WitsmlExplorer.Api.Tests.Workers
 {
     [SuppressMessage("ReSharper", "InconsistentNaming")]
```

Comment:
This doesn't seem to actually do anything? Should probably be removed.
`using System.Diagnostics.CodeAnalysis;` should also be removed.